src/Sulu/Bundle/SecurityBundle/Resources/public/js/components/permission-tab/components/form/main.js

```diff
@@ -0,0 +1,175 @@
+/*
+ * This file is part of the Sulu CMS.
+ *
+ * (c) MASSIVE ART WebServices GmbH
+ *
+ * This source file is subject to the MIT license that is bundled
+ * with this source code in the file LICENSE.
+ */
+
+define(['config', 'sulusecurity/collections/roles'], function(Config, Roles) {
+
+    'use strict';
+
+    var permissionUrl = '/admin/api/permissions',
```

Comment:
Not related. But we should really define all of these routes in Symfony and pass them as options.